app/javascript/mastodon/locales/en.json

```
@@ -206,6 +206,7 @@
   "navigation_bar.pins": "Pinned toots",
   "navigation_bar.preferences": "Preferences",
   "navigation_bar.public_timeline": "Federated timeline",
+  "navigation_bar.faq": "Im@stdon FAQ",
```

react-intelのデフォルトメッセージ  https://github.com/imas/mastodon/blob/imastodon/app/javascript/mastodon/features/getting_started/index.js#L30 と相違してしまっています。
このままだと文言変更になってしまうので合わせたいです。
文言変更が必要だと思う場合には別PRで出してください。
```suggestion
  "navigation_bar.faq": "FAQ(Only available in Japanese)",
```